app/feature/feature-home/src/main/kotlin/com/hedvig/android/feature/home/home/ui/ToolbarIcons.kt

```diff
@@ -52,14 +60,25 @@ fun ToolbarFirstVetIcon(onClick: () -> Unit, modifier: Modifier = Modifier) {
 
 @Composable
 fun ToolbarCrossSellsIcon(onClick: () -> Unit, modifier: Modifier = Modifier) {
+  var isRotated by remember { mutableStateOf(false) }
+  val fullRotation by animateFloatAsState(
+    targetValue = if (isRotated) 360f else 0f,
+    animationSpec = tween(1500, 50),
+  )
+  LaunchedEffect(Unit) {
+    isRotated = true
+  }
```

Comment:
```suggestion
  val isRotated by produceState(false) { value = true }
  val fullRotation by animateFloatAsState(
    targetValue = if (isRotated) 360f else 0f,
    animationSpec = tween(1500, 50),
  )
```
I wonder, does this behave the same way here?
And if yes, do you prefer it or find it less nice to look at?